build.sbt

```diff
@@ -26,6 +27,7 @@ libraryDependencies  ++=  Seq(
   "com.fasterxml.jackson.core"       % "jackson-databind"          % "2.9.0",
   "com.fasterxml.jackson.module"    %% "jackson-module-scala"      % "2.9.0",
   "org.typelevel"                   %% "cats-core"                 % catsVersion,
+  "com.github.julien-truffaut"      %% "monocle-core"              % monocleVersion,
```

Comment:
ahaha, you beat me to it!

Reply:
According to documentation: 
> 1.5.0-cats based on cats 1.0.x

Reply:
as in: `1.5.0-cats` is a retro-compatible version, while `1.5.0` is based on the latest cats?